Cargo.toml

```
@@ -22,6 +22,7 @@ slice-group-by = "0.3.0"
 unicode-segmentation = "1.6.0"
 whatlang = "0.16.1"
 lindera = { version = "=0.16.0", features = ["ipadic"], optional = true }
+pinyin = "0.9"
```

Could you please:
1) keep only the features we need
```suggestion
pinyin = { version = "0.9", default-features = false, features = ["with_tone"], optional = true }
```
2) remove the upper `character_converter` dependency, it is no more needed 😄
3) replace `"dep:character_converter"` by `"dep:pinyin"` in the below `chinese` feature